source/postprocess/dynamic_topography.cc

```diff
@@ -257,7 +257,9 @@ namespace aspect
                              "in the outputted data file (not visualization). "
                              "'true' subtracts the mean, 'false' leaves "
                              "the calculated dynamic topography as is. ");
-
+          prm.declare_entry ("Density above","0",
+                             Patterns::Double (0),
+                             "The density value out of the surface boundary.");
```

Comment:
Sure. Here are the new ones with detailed parameter explanation. I've talked with Rene @gassmoeller  last week on this. The current ASPECT geodynamic topo calculation defaults the outside density to 0 (air). But actually the equation should use the surface density contrast, especially when we want to compute it underneath ocean it needs to be sea water density. More detail can be found here:  https://geodynamics.org/cig/files/8914/1158/8914/CIG_Poster_dyntopo.pdf